### PR TITLE
Mac/OSX user support.

### PR DESCRIPTION
**Pull request by Zzoey**

### CODE REVIEW

**Comment by KartikKannapur** [comment]
The installation steps look good. I have tested the same on Mac/OSX and this pull request can be merged in with the master branch.

**Comment by swapagarwal** [comment]
Cool, thanks @KartikKannapur!
@greyshadows [Prettify](https://github.com/greyshadows/JARVIS-on-Messenger/blob/master/README.md) the steps and I'll merge this asap.

File: README.md [hunk 1]
```diff
@@ -70,11 +70,18 @@ More examples can be found [here](https://github.com/swapagarwal/JARVIS-on-Messe
 
 ### Local Development / Testing
 
+
```
Comment (swapagarwal) [inline README.md]: remove the extra line

File: README.md [hunk 2]
```diff
@@ -70,11 +70,18 @@ More examples can be found [here](https://github.com/swapagarwal/JARVIS-on-Messe
...
 1. Clone this repo.
-2. `sudo apt-get install python-dev libffi-dev libssl-dev`
-3. `pip install -r requirements.txt`
-4. `python jarvis.py`
-5. Visit the following URLs to see results:  
+2. For LINUX users - `sudo apt-get install python-dev libffi-dev libssl-dev`
```
Comment (swapagarwal) [inline README.md]: make this 2a

File: README.md [hunk 3]
```diff
@@ -70,11 +70,18 @@ More examples can be found [here](https://github.com/swapagarwal/JARVIS-on-Messe
...
+3. For Mac/OSX users
```
Comment (swapagarwal) [inline README.md]: make this 2b

File: README.md [hunk 4]
```diff
@@ -70,11 +70,18 @@ More examples can be found [here](https://github.com/swapagarwal/JARVIS-on-Messe
...
+a) If you install Python using brew, the relevant headers are already installed for you. In other words, you don't need python-devel.
+b) `brew install pkg-config libffi`
+`export PKG_CONFIG_PATH=/usr/local/Cellar/libffi/3.0.13/lib/pkgconfig/` # May change with libffi version
+`pip install cffi`
+c) `brew install libtins`
+4. `pip install -r requirements.txt`
```
Comment (swapagarwal) [inline README.md]: the rest steps need not be changed (3,4,5)

**Comment by swapagarwal** [comment]
Hey @greyshadows, will you be willing to complete this? :smile:

**Comment by swapagarwal** [comment]
Merged in 73f6d64991c1dc28824a586c09e1a1ee10e0558f
Thanks @Zzoey! 👍

**Pull request closed without merging**